types/mydecimal.go

```diff
@@ -1084,12 +1085,35 @@ func (d *MyDecimal) FromFloat64(f float64) error {
 }
 
 // ToFloat64 converts decimal to float64 value.
-func (d *MyDecimal) ToFloat64() (float64, error) {
-	f, err := strconv.ParseFloat(d.String(), 64)
-	if err != nil {
-		err = ErrOverflow
+func (d *MyDecimal) ToFloat64() (f float64, err error) {
+	digitsInt := int(d.digitsInt)
```

Comment:
I am more worried about compatibility issues. Do we have a better way to cover this issue.